scripts/circle-ci/check-only-docs-changes.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Returns true if only the docs have been modified
+
+set -x
+
+DOCS_REPO="docs/"
+
+FILES_CHANGED=$(git log --name-only --pretty=format: origin/develop..HEAD)
```

Comment:
This should will still work regardless of changes in develop, it's all references from HEAD back to devleop.  Changed it anyways to a diff using ...